roles/build/tasks/main.yml

```diff
@@ -49,6 +49,17 @@
     src: kubeconfig.j2
     dest: "{{ ansible_env.HOME }}/.kube/config"
 
+- name: Deploy script for cockroach databases/users
```

Comment:
This is renew step only (due to time limit) it contains another data related with cicd.
Next step will be as separate roles for build machine and for cicd/database settings